data/src/main/java/me/androidbox/data/alarm_manager/AlarmReceiver.kt

```diff
@@ -25,15 +32,26 @@ class AlarmReceiver : BroadcastReceiver() {
             }
         }
     }
-    
+
     private fun createAndShowNotification(context: Context, agendaId: String, title: String, description: String) {
         val notificationManager = getNotificationManager(context)
 
+        /** TODO Only Events for the moment, tasks and reminders coming soon... */
+        val agendaIntent = Intent(
+            Intent.ACTION_VIEW,
+            "$EVENT_DEEPLINK={$agendaId}".toUri())
+
+        val pendingIntent = TaskStackBuilder.create(context).run {
+            this.addNextIntentWithParentStack(agendaIntent)
+            this.getPendingIntent(101, PendingIntent.FLAG_IMMUTABLE or PendingIntent.FLAG_CANCEL_CURRENT)
```

Comment:
For the request code. Can I use any number for this? I just added 101

Reply:
Mmmm I think it shouldnt really matter in this case, but i'd still recommend having something like the hashcode

Mostly because I dont know what might happen if you receive 2 notifications from different events. No idea what can happen tho the notification click there